src/platform/tls_openssl.c

```diff
@@ -1906,13 +1920,14 @@ QuicPacketKeyUpdate(
     return Status;
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
 QUIC_STATUS
 CxPlatKeyCreate(
     _In_ CXPLAT_AEAD_TYPE AeadType,
     _When_(AeadType == CXPLAT_AEAD_AES_128_GCM, _In_reads_(16))
     _When_(AeadType == CXPLAT_AEAD_AES_256_GCM, _In_reads_(32))
     _When_(AeadType == CXPLAT_AEAD_CHACHA20_POLY1305, _In_reads_(32))
-        const uint8_t* const RawKey,
+    const uint8_t* const RawKey,
```

Comment:
Undo. This is the style we use in this code.

Reply:
Same for all changes below.

Reply:
These were all just the editor screwing up the copy paste. I'll fix.